### PR TITLE
Server patch 2

### PR DESCRIPTION
**Pull request by DevonRobitaille**

### CODE REVIEW

**Comment by ahogerland** [comment]
Can you add a placeholder `.env` file?

**Comment by DevonRobitaille** [comment]
> Can you add a placeholder `.env` file?

It would appear that github won't let me, however in the readme I am including instructions on what to include.

File: Frontend/src/components/InputComponent/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                        secret: '',
+                    }}
+                    onSubmit={async (values) => {
+                        await submitAttempt(values)
+                    }}
+                >
+                    <Form>            
+            
+                        <div class="attempt-data push">
+                            <label for="secret">Secret</label>
```
Comment (ahogerland) [inline Frontend/src/components/InputComponent/index.js]: Maybe rename this to "Ciphertext" or something. "Secret" makes it seem like some kind of key goes here

**Review by ahogerland — CHANGES_REQUESTED** [review]

File: Frontend/src/components/InputComponent/index.js (hunk 1 above)
Comment (DevonRobitaille) [reply Frontend/src/components/InputComponent/index.js]: I can make that change

**Pull request merged**